src/converter/jfr2pprof.java

```diff
@@ -56,6 +57,29 @@ public boolean equals(final Object other) {
         }
     }
 
+    public static final class Location {
+        final Method method;
+        final int line;
+
+        public Location(final Method method, final int line) {
+            this.method = method;
+            this.line = line;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            Location location = (Location) o;
+            return line == location.line && Objects.equals(method, location.method);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(method, line);
```

Comment:
`Objects.hash` is inefficient because of varargs - please rewrite the method without `Objects.hash`. As a general rule, `hashCode` should not allocate.

Reply:
Thanks. Rewritten.